scripts/characters/names.gd

```diff
@@ -10,7 +10,42 @@ var names = {
agyra","Inakasys","Inakini","Inalys","Inamila","Inapine","Inaroe","Inassa","Inatra","Inayaris","Inazyre","Isodira","Isofyx","Isogani","Isohani","Isokasys","Isokini","Isomiphi","Isonove","Isophina","Isowyn","Isoxi","Jelceli","Jelfyx","Jelli","Jellin","Jelna","Jelniana","Jeltra","Jelwyn","Jelwyse","Jobi","Jobys","Joci","Jofi","Jogani","Jokasys","Jolin","Jomyn","Jona","Jonoa","Joqaryn","Jorhana","Jowyn","Joza","Klobys","Klofyx","Kloli","Klomiphi","Kloniana","Klosany","Krikini","Krilin","Krilys","Kriphina","Kripine","Kriqaryn","Kriqys","Krixis","Lillin","Lilpine","Lilqaryn","Lilrhana","Lilsany","Lilwyn","Lilxi","Lorigani","Lorikini","Lorila","Lorilys","Lorinove","Loriroe","Loriwyn","Loriyaris","Loriyore","Mindira","Minpine","Minsany","Minsys","Mintina","Minwyse","Minxis","Minza","Mybys","Myfi","Myla","Mymiphi","Mypine","Myssa","Myxis","Myyore","Nidysa","Nigyra","Nilin","Niqys","Nisys","Niwyn","Niyore","Nizyre","Nydysa","Nyfi","Nyhana","Nykasys","Nylin","Nylys","Nyna","Nynoa","Nynove","Nypine","Nysany","Odabys","Odadira","Odagyra","Odana","Odanoa","Odanove","Odasys","Odatra","Odaxis","Orbys","Orhana","Orlys","Orpine","Orqys","Orrhana","Orroe","Orsany","Ortra","Orwyn","Phiceli","Phimyra","Phinoa","Phisany","Phissa","Phiwyse","Phiyaris","Phiza","Pridira","Prikini","Priqaryn","Prirhana","Pritina","Pritra","Priwyse","Qibys","Qidira","Qifyx","Qihana","Qili","Qirhana","Qisany","Qissa","Qiyore","Quebi","Queceli","Quegyra","Quehani","Queroe","Quesys","Quetra","Rebi","Rehana","Rekasys","Relys","Remiphi","Rena","Reniana","Renoa","Rexi","Reyaris","Rosigyra","Rosimila","Rosimyra","Rosina","Rosinoa","Rosinove","Rosiroe","Rosisany","Rosiza","Sabys","Samila","Saniana","Sanoa","Sanove","Satra","Seldysa","Sellys","Selmyn","Selnoa","Selroe","Selsys","Selxi","Selzyre","Spilys","Spimiphi","Spinoa","Spipine","Spiqys","Spiyore","Taci","Tagyra","Talys","Tamiphi","Tamyra","Taqaryn","Taqys","Tassa","Tawyse","Taza","Tazyre","Tifadysa","Tifakini","Tifala","Tifamila","Tifawyse","Tifaxi","Tridira","Trifi","Trigani","Triqaryn","Trissa","Trisys","Triyaris","Ufefyx","Ufegani","Ufenoa","Ufephina","Ufepine","Ufeqys","Ufetina","Ufexi","Urici","Urihani","Urili","Urilys","Urina","Uriqys","Uriroe","Urisany","Urissa","Urisys","Uriyore","Venbys","Venceli","Venci","Vengani","Venli","Venpine","Vobys","Vofyx","Vohana","Vokasys","Vola","Voli","Vona","Vophina","Vorhana","Vossa","Votina","Vowyn","Welbi","Welci","Welgani","Welhana","Welkasys","Wellin","Welmyn","Welna","Welpine","Welqys","Weltra","Wrola","Wromyn","Wroniana","Wropine","Wrosys","Wrowyn","Wroza","Xabi","Xabys","Xaceli","Xakasys","Xaphina","Xaroe","Xayaris","Xyrobi","Xyrogyra","Xyroli","Xyrolin","Xyroniana","Xyroqaryn","Xyrorhana","Xyroroe","Xyrosys","Xyrotina","Xyrowyn","Xyrozyre","Ylonove","Ylophina","Yloqys","Ylotina","Ylozyre","Yobys","Yolin","Yomiphi","Yorhana","Yoza","Zanifyx","Zanigani","Zanihana","Zanikini","Zanimyra","Zanisany","Zaniyaris","Zindysa","Zinhani","Zinla","Zinmila","Zinniana","Zinqaryn","Zintina","Zinwyn","Zinxi"],
```

Comment:
Credit to Whims on these in the discord, I'll make sure he gets that credit in the changelog.